backend/aka/tests/test_base.py

```diff
@@ -1,16 +1,10 @@
-from django.core.exceptions import ValidationError
```

Comment:
rename the file or delete. It is not clear what you are testing. It seems like you are testing pythons json parser...